Documentation/data-cleanup/dashboard_resource_helper.py

```
@@ -0,0 +1,147 @@
+        raise Exception(
+            f"Failed to fetch permission groups ({resp.status_code}): {resp.text}"
+        )
+
+    def delete_permission_group(self, group_id: int):
+        url = f"{self.metabase_url}/permissions/group/{group_id}"
+        headers = {
+            "Content-Type": "application/json",
+            "X-Metabase-Session": self.get_session_token(),
+        }
+
+        resp = requests.delete(url, headers=headers)
```

Ruff (0.14.5)

Probable use of `requests` call without timeout

(S113)